pkg/services/store/config.go

```diff
@@ -99,11 +99,13 @@ func (c *GlobalStorageConfig) save() error {
 }
 
 type RootStorageConfig struct {
-	Type        string `json:"type"`
-	Prefix      string `json:"prefix"`
-	Name        string `json:"name"`
-	Description string `json:"description"`
-	Disabled    bool   `json:"disabled,omitempty"`
+	Type             string `json:"type"`
+	Prefix           string `json:"prefix"`
+	MountPoint       string `json:"mountPoint"`
```

Comment:
for now, can we just make everything that is configured live under "content"?

Later we could look at how to make a more generic symlink system, but let's cross that bridge when we get there :)